### PR TITLE
fix: make FIR searchable by its identifier

### LINKED ISSUE #218: Search by FIR code
When editing a users profile, you can only search for FIR name, searching by FIR code would make this a lot easier.

### PR DESCRIPTION
**Pull request by SamHep0803**
- When attaching an FIR to a user, the FIR can now be searched by its four-character identifier instead of its full name. (Closes #218)

### CODE REVIEW

**Review by daveroverts — CHANGES_REQUESTED** [review]
Hi, the solution isn't complete. Now you're showing just the `identifier`. What we really need is to search/show `identifier` + `name`. Example: `EHAA | Amsterdam FIR`.

I already implemented this in `FlowMeasureResource`, here are 2 examples:

<https://github.com/ECFMP/flow/blob/95fa1660a398b1377ae1360e32c4886041a778d3/app/Filament/Resources/FlowMeasureResource.php#L45-L48>

https://github.com/ECFMP/flow/blob/95fa1660a398b1377ae1360e32c4886041a778d3/app/Filament/Resources/FlowMeasureResource.php#L227-L236

**Comment by SamHep0803** [comment]
Changes added! :)

File: app/Filament/Resources/UserResource/RelationManagers/FlightInformationRegionsRelationManager.php [hunk 1]
```diff
@@ -45,7 +52,18 @@ public static function table(Table $table): Table
             ])
             ->headerActions([
                 Tables\Actions\CreateAction::make(),
-                Tables\Actions\AttachAction::make(),
+                Tables\Actions\AttachAction::make()
+                    ->label("Add User to FIR")
```
Comment (daveroverts) [inline app/Filament/Resources/UserResource/RelationManagers/FlightInformationRegionsRelationManager.php:56]: Can this label be removed? I'm pretty sure we can live with `Attach` (default)

**Review by daveroverts — CHANGES_REQUESTED** [review]

File: app/Filament/Resources/UserResource/RelationManagers/FlightInformationRegionsRelationManager.php (hunk 1 above)
Comment (SamHep0803) [reply app/Filament/Resources/UserResource/RelationManagers/FlightInformationRegionsRelationManager.php:56]: Apologies, left it in when I was internally testing.

**Review by daveroverts — APPROVED** [review]

**Review by daveroverts — CHANGES_REQUESTED** [review]
Actually, one more thing. Can you run Pint (using `./vendor/bin/pint`). It doesn't matter if you commit files you haven't edited.

**Review by daveroverts — APPROVED** [review]

**Pull request merged**